cmd/auth/login.go

```diff
@@ -34,7 +34,47 @@ const minimalDbConnectVersion = "13.1"
 func newLoginCommand(persistentAuth *auth.PersistentAuth) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "login [HOST]",
-		Short: "Authenticate this machine",
+		Short: "Log into a Databricks workspace or account",
+		Long: `Log into a Databricks workspace or account.
+This command logs you into the Databricks workspace or account and saves
+the authentication configuration in a profile (in ~/.databrickscfg by default).
+
+This profile can then be used to authenticate other Databricks CLI commands by
+specifying the --profile flag. This profile can also be used to authenticate
+other Databricks tooling that supports the Databricks Unified Authentication
+Spec. This includes the Databricks Go, Python and Java SDKs. For more information,
+you can refer to the documentation linked below.
+  AWS: https://docs.databricks.com/en/dev-tools/auth/index.html
```

Comment:
I think you can leave out the `en` or `en-us` in these URLs, which should enable the user's localized help preference to control.

Reply:
Thanks, done!

Reply:
Done, thanks!